packages/web/src/components/earn/incentivized-pool-card-list/IncentivizedPoolCardList.tsx

```diff
@@ -7,55 +7,54 @@ import { type PoolListProps } from "@containers/incentivized-pool-card-list-cont
 import IncentivizedPoolCard from "@components/earn/incentivized-pool-card/IncentivizedPoolCard";
 import { SHAPE_TYPES, skeletonStyle } from "@constants/skeleton.constant";
 import LoadMoreButton from "@components/common/load-more-button/LoadMoreButton";
-
 interface IncentivizedPoolCardListProps {
   list: Array<PoolListProps>;
   loadMore?: boolean;
   isFetched: boolean;
   onClickLoadMore?: () => void;
-  windowSize: number;
   currentIndex: number;
+  routeItem: (id: number) => void;
+  windowSize: number;
 }
 
 const IncentivizedPoolCardList: React.FC<IncentivizedPoolCardListProps> = ({
   list,
   loadMore,
   isFetched,
   onClickLoadMore,
-  windowSize,
   currentIndex,
-}) => {
-  return (
-    <IncentivizedWrapper>
-      <PoolListWrapper>
-        {isFetched &&
-          list.length > 0 &&
-          list.map((item, idx) => (
-            <IncentivizedPoolCard item={item} key={idx} />
-          ))}
-        {!isFetched &&
-          Array.from({ length: 8 }).map((_, idx) => (
-            <span
-              key={idx}
-              className="card-skeleton"
-              css={skeletonStyle("100%", SHAPE_TYPES.ROUNDED_SQUARE)}
-            />
-          ))}
-      </PoolListWrapper>
-      {windowSize > 1000 ? (
-        loadMore &&
-        onClickLoadMore && (
-          <LoadMoreButton show={loadMore} onClick={onClickLoadMore} />
-        )
-      ) : (
-        <div className="box-indicator">
-          <span className="current-page">{currentIndex}</span>
-          <span>/</span>
-          <span>{list.length}</span>
-        </div>
-      )}
-    </IncentivizedWrapper>
-  );
-};
+  routeItem,
+  windowSize,
+}) => (
+  <IncentivizedWrapper>
+    <PoolListWrapper>
+      {isFetched &&
+        list.length > 0 &&
+        list.map((item, idx) => (
+          <IncentivizedPoolCard item={item} key={idx} routeItem={routeItem} />
+        ))}
+      {!isFetched &&
+        Array.from({ length: 8 }).map((_, idx) => (
+          <span
+            key={idx}
+            className="card-skeleton"
+            css={skeletonStyle("100%", SHAPE_TYPES.ROUNDED_SQUARE)}
+          />
+        ))}
+    </PoolListWrapper>
+    {windowSize > 1000 ? (
```

Comment:
Showing components by comparing their windowSize is render intensive.
It's better to use conditional rendering with a type or boolean that doesn't change much in value.

Reply:
Good idea. I fixed it by rendering with boolean value.